reference/configuration/framework.rst

```diff
@@ -1776,6 +1777,18 @@ default_redis_provider
 The DSN to use by the Redis provider. The provider is available as the ``cache.redis``
 service.
 
+default_memcached_provider
+..........................
+
+.. versionadded:: 3.3
+
```

Comment:
This empty line should be removed (versionadded is kinda a strange directive that requires this...)

Reply:
Done.